lib/puppet/functions/get_rsan_ip.pp

```diff
@@ -0,0 +1,14 @@
+function Functions::get_rsan_ip() {
+  if $settings::storeconfigs {
+    $rsan_ip =
+                puppetdb_query('resources[network] {
+                    type = "Class" and
+                    title = "Rsan::Importer" and
+                    nodes {
+                      deactivated is null and
+                      expired is null
+                    }
+                  }').map |$data| { $data['certname'] }
```

Comment:
Has this been tested? It'll return a list of certnames right, not a list of IPs? You could test it at the command line by running:

```shell
puppet query 'resources[network] { type = "Class" and title = "Rsan::Importer" and nodes { deactivated is null and expired is null } }'
```

That would show us the output that the `map()` function would then be operating on